### ISSUE TITLE
Desktop version for the app

### ISSUE THREAD

**Issue by parthvyas7** [davidhealey/waistline#617]
It would be great if the desktop version of the app would initiate its development as in the market none of the calorie tracker apps offers functionality other than that in the mobile version and also managing food, recipes & meals database on the desktop is much easier in the mobile device.

**Comment by davidhealey** [comment]
I don't quite understand what you are asking, could you rephrase the question.

**Comment by parthvyas7** [comment]
@davidhealey I mean to say that it would be great if there was a desktop version of the app, as mass managing the food, recipes & meals database on the desktop is much easier on the mobile device.

**Comment by davidhealey** [comment]
Aha ok. This is has already been requested for some time https://github.com/davidhealey/waistline/issues/211

I briefly attempted to make an Electron/Desktop build but couldn't get it running.

**Comment by EmilJunker** [comment]
The main issue is the data synchronization between the clients. Without proper synchronization, what's the point of having a desktop version? Waistline is designed to keep all data stored locally, so implementing a reliable sync system would be very difficult.

**Comment by kquote03** [comment]
Is it possible to put all the local data on a normal folder / file and just have an external sync program like syncthing sync that?

From my experience syncthing has been super reliable on Android and PC.

**Comment by alexanderwiller** [comment]
I would like to +1 this, the stated way of doing things (e.g. as with KeePass) is exemplary for a class of file storage-based "local" apps that can still be used on multiple machines easily, by means of a separate file syncing service.

It would be a massive functionality improvement in my opinion if the app could be run in some electron environment on desktop systems.

**Comment by nabeelr** [comment]
Yeah I agree with @kquote03 about adding an option to move the database to somewhere where an app like Syncthing can grab it.

This would also have the side effect of enabling automatic routine backups of your Waistline DB, for those of us, like myself that have 4 years of data in our DBs and are tracking long term trends for diet and nutrition.
